src/dirs.proj

```diff
@@ -4,6 +4,7 @@
 
   <ItemGroup>
     <Project Include="*\**\*.csproj" />
+    <Project Include="*\**\*.depproj" />
```

Comment:
I don't think you need the depproj here.  Just update the version of buildtools used in this repo and use the CopyNugetImplementations property in the csproj.